samcli/commands/init/init_templates.py

```diff
@@ -26,6 +27,7 @@
 MANIFEST_URL = "https://raw.githubusercontent.com/aws/aws-sam-cli-app-templates/master/manifest-v2.json"
```

Comment:
We need to update here where we only pull the `manifest.json` file. Otherwise it will lead to error where a template will be listed since this will download from develop HEAD, but when we clone by using this hash, it will be out of date.

Reply:
Good catch! Will update the `mainfest.json` url as well.